### PR TITLE
Review and refine for ros2 branch

### PR DESCRIPTION
**Pull request by ChloeWu1**
Signed-off-by: Wu, Jiawei [jiawei.wu@intel.com](mailto:jiawei.wu@intel.com)

### CODE REVIEW

**Comment by gitpqLee** [comment]
@LewisLiuPub  Could please help to review this PR? which update docs. Thx

**Comment by LewisLiuPub** [comment]
branch foxy keeps for foxy and openvino 2021.4LTS, so the related doc should also be kept.
branch dashing is deprecated, but we will keep it for a span of time, so the related doc also needs to keep.

**Review by LewisLiuPub — COMMENTED** [review]
please check the new review comments.

File: README.md [hunk 1]
```diff
@@ -1,46 +1,170 @@
...
+	* [Inference Implementations](#inference-implementations)
+	* [ROS Interfaces and Outputs](#ros-interfaces-and-outputs)
+	* [Demo Result Snapshots](#demo-result-snapshots)
+* [➤ Installation & Launching](#installation-and-launching)
+* [➤ Reference](#reference)
 
-* [x] ROS2 Dashing
-* [x] ROS2 Eloquent
+# Introduction
+## ROS2 Version Supported
```
[the remark this answers is not in the document]
Comment (ChloeWu1) [reply README.md:25]: Matrix added.

File: README.md [hunk 2]
```diff
@@ -1,46 +1,170 @@
...
 * [x] Object Detection
 * [x] Face Detection
-* [x] Age-Gender Recognition
+* [x] Age Gender Recognition
 * [x] Emotion Recognition
 * [x] Head Pose Estimation
 * [x] Object Segmentation
 * [x] Person Re-Identification
 * [x] Vehicle Attribute Detection
 * [x] Vehicle License Plate Detection
```
[the remark this answers is not in the document]
Comment (ChloeWu1) [reply README.md:48]: Table of models we supported and tested has been added.

[the remark this answers is not in the document]
Comment (ChloeWu1) [reply README.md:48]: Updated.

|Branch Name|ROS2 Version Supported|Openvino Version|OS Version|
|-----------------------|-----------------------|--------------------------------|----------------------|
|[ros2](https://github.com/intel/ros2_openvino_toolkit/tree/ros2)|Galactic, Foxy, Humble|V2022.1, V2022.2|Ubuntu 20.04, Ubuntu 22.04|
|[dashing](https://github.com/intel/ros2_openvino_toolkit/tree/dashing)|Dashing|V2022.1, V2022.2|Ubuntu 18.04|
|[foxy-ov2021.4](https://github.com/intel/ros2_openvino_toolkit/tree/foxy)|Foxy|V2021.4|Ubuntu 20.04|
|[galactic-ov2021.4](https://github.com/intel/ros2_openvino_toolkit/tree/galactic-ov2021.4)|Galactic|V2021.4|Ubuntu 20.04|

File: README.md [hunk 3]
```diff
@@ -1,49 +1,258 @@
...
+
+<p>
+<details>
+<summary>Input Resource Table</summary>
+
+|Input Resource|Description|
+|--------------------|------------------------------------------------------------------|
+|StandardCamera|Any RGB camera with USB port supporting. Currently only the first USB camera if many are connected.|
+|RealSenseCamera| Intel RealSense RGB-D Camera, directly calling RealSense Camera via librealsense plugin of openCV.|
+|RealSenseCameraTopic| Any ROS topic which is structured in image message.|
```
[the remark this answers is not in the document]
Comment (ChloeWu1) [reply README.md:138]: Renamed.

File: README.md [hunk 4]
```diff
@@ -1,49 +1,258 @@
...
+|Vehicle and License Detection| Vehicle and license detection based on Intel models.|[pipeline_vehicle_detection.yaml](./sample/param/pipeline_vehicle_detection.yaml)|[vehicle-license-plate-detection-barrier-0106](https://github.com/openvinotoolkit/open_model_zoo/tree/master/models/intel/vehicle-license-plate-detection-barrier-0106)<br>[vehicle-attributes-recognition-barrier-0039](https://github.com/openvinotoolkit/open_model_zoo/tree/master/models/intel/vehicle-attributes-recognition-barrier-0039)<br>[license-plate-recognition-barrier-0001](https://github.com/openvinotoolkit/open_model_zoo/tree/master/models/intel/license-plate-recognition-barrier-0001)|
+|Object Segmentation| Object segmentation.|[pipeline_segmentation.yaml](./sample/param/pipeline_segmentation.yaml)<br>[pipeline_segmentation_image.yaml](./sample/param/pipeline_segmentation_image.yaml)<br>[pipeline_video.yaml](./sample/param/pipeline_video.yaml)|[semantic-segmentation-adas-0001](https://github.com/openvinotoolkit/open_model_zoo/tree/master/models/intel/semantic-segmentation-adas-0001)<br>[deeplabv3](https://github.com/openvinotoolkit/open_model_zoo/tree/master/models/public/deeplabv3)|
```
[the remark this answers is not in the document]
Comment (ChloeWu1) [reply README.md:160]: Added.

File: README.md [hunk 5]
```diff
@@ -1,49 +1,258 @@
...
+
+## Inference Implementations
+Currently, the corresponding relation of supported inference features, models used and yaml configurations are listed as follows:
+
+<p>
+<details>
+<summary>Inference Feature Correspondence Table</summary>
+
+|Inference|Description|YAML Configuration|Model Used|
+|-----------------------|------------------------------------------------------------------|----------------------|----------------------|
```
[the remark this answers is not in the document]
Comment (ChloeWu1) [reply README.md:153]: Fixed.

**Comment by LewisLiuPub** [comment]
LGTM.

**Pull request merged**